engine/api/workflow/artifact.go

```diff
@@ -0,0 +1,44 @@
+package workflow
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/engine/api/services"
+	"github.com/ovh/cds/sdk"
+)
+
+func ListArtifacts(ctx context.Context, db gorp.SqlExecutor, runID int64) (sdk.CDNItemLinks, error) {
```

Comment:
Maybe It could be nice to have a `cdn` package in the API that expose all its business functions at place. It could be easily mockable in unit tests